tectonic/dpx-pdfobj.c

```diff
@@ -1693,8 +1693,9 @@ apply_filter_TIFF2_1_2_4 (unsigned char *raster,
                 }
             }
         }
-        if (outbits > 0)
+        if (outbits > 0) {
             raster[k] = (outbuf << (8 - outbits)); k++;
```

Comment:
In this case `k++;` can be safely removed because the variable only ever gets [set](https://github.com/ronnychevalier/tectonic/blob/ababa0acbb2faad2314857ee3b318492c6b883a0/tectonic/dpx-pdfobj.c#L1671) after the increment operation.